### DISCUSSION TITLE
Can we add comments to a review from within the Web-Based Editor?

### DISCUSSION THREAD

**Discussion by bpartridge** [community/community#5451] [Codespaces]
The new web-based editor https://docs.github.com/en/codespaces/developing-in-codespaces/web-based-editor would be transformative for our team in reviewing highly complex pull requests!

But I can't figure out how to actually add a new PR comment to a line - only global comments or responses to comments already made in the normal PR review interface. There doesn't seem to be anything about adding a comment in the command palette or in the context menu.

See attached screenshot - ideally there would be something that allows us to create a comment on the selected lines from there. Am I missing something?

<img width="1180" alt="Screen Shot 2021-09-03 at 1 51 10 PM" src="https://user-images.githubusercontent.com/20748/132047608-aa659b74-1b8e-431f-bec3-644f5b3f8ebc.png">

**Comment by AravindGopala** [comment]
I am able add comments in the web based editor.
These are my steps, hope it helps:
once a pull request is opened in the web-based editor , 
when I click on the white bar next to the line number, I see a drop down to start discussion/comment on each line/lines of code.
![image](https://user-images.githubusercontent.com/7443625/132108812-d0909445-0622-42c4-b62d-f57b6637ab65.png)
